worksheet_D/oxo.py

```diff
@@ -0,0 +1,107 @@
+class OxoBoard:
+    def __init__(self):
+        # Initialising board array
+        self.boardList = [[0, 0, 0], [0, 0, 0], [0, 0, 0]]
+
+    def get_square(self, x, y):
+        # Returns value of board square
+        return self.boardList[y][x]
```

Comment:
Is there a reason you chose to write `[y][x]` instead of `[x][y]`? It doesn't matter too much as long as you're consistent, but it is more usual to write coordinates in x,y order.